src/main/java/Jay.java

```diff
@@ -0,0 +1,153 @@
+    public static void main(String[] args) {
+        Jay jay = new Jay("Jay");
+        jay.start();
+    }
+
+    private final TaskList tasks;
+    private final Ui ui;
+
+    public Jay(String name) {
+        this.tasks = new TaskList("tasks.txt");
+        this.ui = new Ui(name);
+    }
+
+    /**
+     * Starts the chatbot.
+     */
+    public void start() {
+        ui.greet();
+        Scanner scanner = new Scanner(System.in);
+
+        while (scanner.hasNext()) {
+            String command = scanner.nextLine();
+            if (command.equals("bye")) {
+                ui.quit();
+                break;
+            } else {
+               ui.output(this.processCommand(command));
+            }
+        }
+    }
+
+    /**
+     * Processes the command and returns the response.
+     *
+     * @param commandStr The command to be processed.
+     * @return The response to the command. Show to the user by the chatbot.
+     */
+    private String processCommand(String commandStr) {
+        Command command = new Command(commandStr);
+        Command.CommandType commandType = command.getCommandType();
+
+        try {
+            switch (commandType) {
+                case List -> {
+                    return this.showTasks();
+                }
+                case Add -> {
+                    Task.TYPE taskType = command.getTaskType();
+                    return this.addTask(Parser.parseTask(taskType, command));
+                }
+
+                case Mark -> {
+                    int taskNumber = command.getTaskNumber();
+                    Task task = this.tasks.markAsDone(taskNumber);
+                    return "Nice! I've marked this task as done:\n" + task;
+                }
+
+                case Unmark -> {
+                    int taskNumber = command.getTaskNumber();
+                    Task task = this.tasks.markAsNotDone(taskNumber);
+                    return "OK, I've marked this task as not done yet:\n" + task;
+                }
+
+                case Delete -> {
+                    int taskNumber = command.getTaskNumber();
+                    return this.deleteTask(taskNumber);
+                }
+
+                case Find -> {
+                    String keyword = command.getKeyword();
+                    return this.findTasks(keyword);
+                }
+
+                case Exit -> {
+                    return "Bye. Hope to see you again soon!";
+                }
+
+                default ->
+                    throw new InvalidCommandException("OOPS!!! I'm sorry, but I don't know what that means :-(");
+            }
+        } catch (InvalidCommandException | InvalidTaskException | DataIOException e) {
+            return e.getMessage();
+        }
+    }
+
+    /**
+     * Shows the tasks in the task list.
+     * If the task list is empty, a message will be shown to the user.
+     * @return The tasks in the task list or a message indicating that the task list is empty.
+     */
```

Comment:
You did this correctly in the previous methods. There should be an empty line between the description of the method and the list of parameters in the JavaDoc.